src/en/howto-prometheus.md

```diff
@@ -0,0 +1,68 @@
+Title: Configure metrics gathering with Prometheus
+
+#  Configure metrics gathering with Prometheus
+
+
+## Overview
+
+While it is possible to [write charms that collect metrics][developer-metrics], some
```

Comment:
The first two sentences give me the impression that we're not providing something that's better than the existing charm metrics. The new Prometheus integration is orthogonal. Charm metrics are metrics about charms, Prometheus metrics are metrics about Juju itself (operation of controllers and agents).

I think this should be reworded a bit. It's probably worth mentioning/cross-referencing still, but I would like to see mention the distinction made.